src/lib/containers/GenericCard.tsx

```diff
@@ -424,7 +424,7 @@ export default class GenericCard extends React.Component<
       paddingBottom: showFooter || imageFileHandleIdValue ? undefined : '15px',
     }    
     const icon:JSX.Element = <>
-        {imageFileHandleIdValue && <div className="SRC-imageThumbnail" style={{padding: genericCardSchemaDefined.thumbnailRequiresPadding ? '21px' : 'inherit'}}>
+        {imageFileHandleIdValue && <div className="SRC-imageThumbnail" style={{padding: genericCardSchemaDefined.thumbnailRequiresPadding ? '21px' : undefined}}>
```

Comment:
Setting to an undefined value causes the style not to be overridden (which is the intended behavior in this case)